aws/Vault-Single-Deployment.json

```diff
@@ -823,7 +636,7 @@
                                  "'/PrimaryVaultIP',",
                                  "'1.1.1.1',",
                                  "'/DRPassword',",
-                                 "drp,",
+                                 "adminp,",
```

Comment:
Is this correct ? why did the parameter change?

Reply:
In vault single deployment there is no need in dr password. There was an object which was created during deployment to store Admin password in DR section and then used here. I have simplified it to use the same password only here without creating a new object.

Reply:
@maishsk - fixed StorePasswordLambda desciption